services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterTest.java

```diff
@@ -71,11 +74,13 @@ public class MessageWriterTest extends AbstractSdtUnitTestBase {
     /**
      * Set up the variables.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
-        // Nicemock returns default values
-        messageSender = EasyMock.createMock(MessageSender.class);
-        queueConfig = new QueueConfig();
+        MockitoAnnotations.openMocks(this);
```

Comment:
This should be able to be removed as the ExtendWith(MockitoExtension.class) annotation initialises the annotated mocks in the class.

Reply:
removed

Reply:
Confirmed openMocks() line removed.